hledger-lib/Hledger/Utils/String.hs

```diff
@@ -134,10 +134,10 @@ whitespacechars = " \t\n\r"
 redirectchars   = "<>"
 
 escapeDoubleQuotes :: String -> String
-escapeDoubleQuotes = regexReplace "\"" "\""
+escapeDoubleQuotes = id  -- regexReplace "\"" "\""
 
 escapeQuotes :: String -> String
-escapeQuotes = regexReplace "([\"'])" "\\1"
+escapeQuotes = id  -- regexReplace "([\"'])" "\\1"
```

Comment:
This is not used anywhere, but is exported. Should it be removed or kept?

Reply:
I imagine these things could be handy for hledger script authors working with command line arguments, but quoteIfNeeded probably is enough. Sure let's remove it.